### PR TITLE
Big refactors

### PR DESCRIPTION
**Pull request by wastadtlander**
This PR contains: 

1. Refactors to project structure
2. Cleanup
3. Small portion of component to display `not a youtube page` (Going to extract this into its own component at some point)

### CODE REVIEW

File: craco.config.js [hunk 1]
```diff
@@ -10,8 +10,8 @@ module.exports = {
             paths.appIndexJs,
           ].filter(Boolean),
           content: [
-            "./src/chromeServices/content.ts",
-            "./src/chromeServices/DOMEvaluator.ts",
+            "./src/background/content.ts",
+            "./src/services/chromeServices/DOMEvaluator.ts",
           ],
```
Comment (quinnha) [inline craco.config.js:15]: are we going to have more services than chromeServices/domevaluator? if not maybe just moving domevaluator into services could be better

File: package-lock.json [hunk 2]
```diff
@@ -2517,20 +2517,6 @@
         "node": ">=8"
       }
     },
```
Comment (quinnha) [inline package-lock.json:2519]: bring this file back pls

File: public/manifest.json [hunk 3]
```diff
@@ -10,8 +10,8 @@
   "permissions": ["activeTab"],
   "content_scripts": [
     {
-      "matches": ["https://www.youtube.com/watch?v=*"],
-      "js": ["./static/js/content.js"]
+      "js": ["./static/js/content.js"],
+      "matches": ["https://www.youtube.com/watch?v=*"]
```
Comment (quinnha) [inline public/manifest.json:14]: average google intern change

File: src/components/App/App.tsx [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+  //   console.log(window.location);
+  //   setIsOnYouTubeVideo(isYouTubeVideoPage);
+  // }
+
+  // useEffect(() => {
+  //   checkYouTubePage();
+  //   const hashChangeListener = () => checkYouTubePage();
+  //   window.addEventListener("hashchange", hashChangeListener);
+  //   return () => window.removeEventListener("hashchange", hashChangeListener);
+  // })
```
Comment (quinnha) [inline src/components/App/App.tsx]: remove if not needed

File: src/components/App/App.tsx [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+  };
+
+  return (
+    <div>
+      <button onClick={getUILanguage}>console log language</button>
+      {/* {isOnYouTubeVideo ? (
+        <button onClick={getUILanguage}>console log language</button>
+      ) : (
+        <p>This is not a YouTube video page.</p>
+      )} */}
```
Comment (quinnha) [inline src/components/App/App.tsx]: remove if not implemented

File: src/components/Popup/popup.js [hunk 6]
```diff
@@ -1,3 +1,5 @@
+// Translate to .tsx
```
Comment (quinnha) [inline src/components/Popup/popup.js:1]: guessing this is in a later pr?

**Review by quinnha — CHANGES_REQUESTED** [review]

**Comment by quinnha** [comment]
can you also add a line in the README to npm init before npm run build?

File: craco.config.js (hunk 1 above)
Comment (wastadtlander) [reply craco.config.js:15]: Done.

File: public/manifest.json (hunk 3 above)
Comment (wastadtlander) [reply public/manifest.json:14]: alphabet boiiiiiiiiiiiiii

File: package-lock.json (hunk 2 above)
Comment (wastadtlander) [reply package-lock.json:2519]: What do you mean? The file auto updates.

File: src/components/App/App.tsx (hunk 4 above)
Comment (wastadtlander) [reply src/components/App/App.tsx:19]: Leaving for later use, will be needed.

File: src/components/Popup/popup.js (hunk 6 above)
Comment (wastadtlander) [reply src/components/Popup/popup.js:1]: Yeah

Comment (quinnha) [reply src/components/Popup/popup.js:1]: make a issue for it then

**Review by quinnha — APPROVED** [review]
lets get this moneyyy

**Pull request merged**